dojo/settings/settings.dist.py

```diff
@@ -1123,6 +1123,7 @@ def saml2_attrib_map_format(dict):
     'Hydra Scan': ['title', 'description'],
     'DrHeader JSON Importer': ['title', 'description'],
     'PWN SAST': ['title', 'description'],
+    'docker-bench-security Scan': ['title', 'description'],
```

Comment:
I think we should use vuln_id_from_tool here

Reply:
ok, thank you, t think you mean - only vuln_id_from_tool?

Reply:
yep, but I saw you fixed that